src/libs/actions/Session/index.js

```diff
@@ -250,7 +250,7 @@ function signIn(password, twoFactorAuthCode) {
                     Onyx.merge(ONYXKEYS.ACCOUNT, {requiresTwoFactorAuth: true, isLoading: false});
                     return;
                 }
-                Onyx.merge(ONYXKEYS.ACCOUNT, {error: Localize.translateLocal(errorMessage), isLoading: false});
+                Onyx.merge(ONYXKEYS.ACCOUNT, {errors: {[DateUtils.getMicroseconds()]: Localize.translateLocal(errorMessage)}, isLoading: false});
```

Comment:
I have a feeling that this will throw an error and potentially crash the App since `errorMessage` is not a key defined in any of our locales.

Reply:
This is phrase key coming `ErrorUtils.getAuthenticateErrorMessage()`